src/settings.rs

```diff
@@ -100,10 +101,15 @@ impl Settings {
     }
 
     fn validated(self) -> Result<Self, ConfigError> {
+        let lookback_combined = LOOKBACK_RANGE + LOOKBACK_LEEWAY;
         if self.relay.anchor_frequency == 0 {
             Err(ConfigError::InvalidAnchorFrequency)
         } else if self.relay.confirmations >= self.relay.anchor_frequency {
             Err(ConfigError::InvalidConfirmations)
+        } else if self.relay.homechain.interval >= lookback_combined {
```

Comment:
clippy doesn't like this, although I think its fine. might switch to an or instead of having two separate else ifs